### DISCUSSION TITLE
[Fixed] Website/Server Down(?) | Unable to visit Main website

### DISCUSSION THREAD

**Discussion by sherkhan30452** [SubtitleEdit/subtitleedit#8482] [General]
I just wanted to inform @niksedk that I can't reach the main website https://nikse.dk/subtitleedit

![image](https://github.com/SubtitleEdit/subtitleedit/assets/33783004/03078c07-5484-4007-b1ee-78d372d72009)

**Comment by niksedk** [comment]
Sorry, I don't know why... the hosting company seems to have problems...

**Comment by sherkhan30452** [comment]
Fixed
